recipes/lstchain/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "lstchain" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3ea764c3c1c79c748593cc82043ca0dd408d38e563b5015bd890c2a3584d490c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools-scm
+  run:
+    - python >=3.8
+    - astropy =5
```

Comment:
You have many tight dependencies here that are not upstream. This one, for example, is ~=5 in the package. You can use grayskull to create the recipe, it will not only make one according to the metadata but also expand the ~= for you into its easier to read form.

Reply:
But `=5` in conda means  any 5.x version, right? So it should be identical in meaning to `~=5.0` in pip / PEP 440 definition.

Reply:
I'm not sure how that behaves with the <.dev0, I'd rather have the proper expansion there to be explicit.